python/doc/content/python/sqa/python_sll.md

```diff
@@ -1 +1,269 @@
-!template load file=sqa/app_sll.md.template category=python app=MOOSE Tools
+!template load file=sqa/sll.md.template project=MOOSE Tools
+
+## NumPy
```

Comment:
"MOOSE Tools" encompasses mooseutils, MooseDocs, TestHarness and the miscellaneous utilities in the `python` dir. We explicitly leave out Peacock and Chigger for the purposes of SQA, because they're truly optional, so VTK isn't necessary for this list.